packages/contracts/controller/src/account.cairo

```diff
@@ -303,55 +309,39 @@ mod CartridgeAccount {
 
     #[abi(embed_v0)]
     impl UserAccountImpl of IUserAccount<ContractState> {
-        fn change_owner(ref self: ContractState, signer_signature: SignerSignature) {
-            assert(self.is_caller_allowed(get_caller_address()), 'caller-not-owner');
-
-            let new_owner = signer_signature.signer();
-
-            self.assert_valid_new_owner_signature(signer_signature);
-
-            let new_owner_storage_value = new_owner.storage_value();
-            self.write_owner(new_owner_storage_value);
-
-            if let Option::Some(new_owner_pubkey) = new_owner_storage_value
-                .starknet_pubkey_or_none() {
-                self.emit(OwnerChanged { new_owner: new_owner_pubkey });
-            };
-            let new_owner_guid = new_owner_storage_value.into_guid();
-            self.emit(OwnerChangedGuid { new_owner_guid });
-            self.emit(SignerLinked { signer_guid: new_owner_guid, signer: new_owner });
-        }
-
         fn get_owner(self: @ContractState) -> felt252 {
```

Comment:
Yea, i just didn't want to break the interface. 
I wonder maybe we could provide a method to return all owners' guids or is it not necessary?

Reply:
I don't think it is necessary since that information should be easily indexed offchain using events. Imo we should remove this method, since it's not really returning useful information at this point

Reply:
Removed